src/pages/home/home.js

```diff
@@ -5,6 +5,7 @@ import { HorizontalCard } from "../../components/home-product-cards/horizontal-c
 import VerticalCard from "../../components/home-product-cards/vertical-card/VerticalCard";
 import { useProductContext } from "../../contexts/product-context";
 import { Link } from "react-router-dom";
+
```

Comment:
Separating the module import with local imports will increase the readability
Ex:
```
//All module imports at the start
import axios from "axios";

//All local import
import Card from './Card';
```